e2e/browser/test/e2e.playwright.ts

```diff
@@ -0,0 +1,117 @@
+//
+// Copyright 2022 Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import { test, expect } from "@inrupt/internal-playwright-helpers";
+import { getBrowserTestingEnvironment } from "@inrupt/internal-test-env";
+
+// TODO: Add "vcProvider" to TestingEnvironmentBase instead of TestingEnvironmentNode since it is used by both instances.
+const { vcProvider } = getBrowserTestingEnvironment({
+  vcProvider: "",
+});
+test("Issue credential to a resource, then revoking it", async ({
+  page,
+  auth,
+}) => {
+  await auth.login({ allow: true });
+  // eslint-disable-next-line playwright/no-conditional-in-test
+  await page.fill("input[data-testid=vcProvider]", vcProvider || "");
+  // Create the resource
+  await Promise.all([
+    page.click("button[data-testid=create-resource]"),
+    page.waitForRequest((request) => request.method() === "POST"),
+    page.waitForResponse((response) => response.status() === 201),
+  ]);
+  await expect(
+    page.innerText("span[data-testid=resource-iri]")
+  ).resolves.toMatch(/https:\/\/.*\.txt/);
```

Comment:
I wouldn't bother creating a dedicated resource. It makes sense if testing Access Grant capabilities, i.e. "can I use the issued Access Grant to get access to a resource", but all we want to test here is "can I issue a VC passing some data in and checking that said data made it through". The fact that the VC happens to be an Access Grant is not that important, as the Access Grant features are tested in the dedicated library.

That means this step can be removed, as well as the related components in the test app.

Reply:
```suggestion
```